pkg/Library/Mediapipe/FaceSticker.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
```

Comment:
does this replace the `HolisticSticker.js`? should we remove that one?

Reply:
I thought we should wait and see. They are distinct.